views/index.ejs

```diff
@@ -8,10 +8,13 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length; %> <!-- Print the number of items in the cart here --></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
+     <% cart.forEach(function(product){ %>
+       <li><%= product.name %></li>
 
+     <% }); %>
       <!-- If you're adventurous, try adding a delete form/button for each item to remove it from the cart! -->
         <!-- Note: if you try the delete form, you'll probably need a line like this inside your form: -->
         <!-- <input type="hidden" name="_method" value="delete"> -->
```

Comment:
Overall great job!! Appreciate your indenting and spacing consistency.. makes for some clean code!